# WaypointVector.msg

Header header
Waypoint[] waypoints
